tests/unittest_modutils.py

```diff
@@ -57,7 +57,7 @@ def test_find_egg_module(self) -> None:
 
 
 class LoadModuleFromNameTest(unittest.TestCase):
-    """load a python module from it's name"""
+    """load a python module from its name"""
 
```

Comment:
I think it would be nice to have a test mocking ``importlib.import_module`` with a check of capsys stdout and stderr being falsey and caplog outputting what is expected when set in info level.

```python
def test_import_dotted_library(
    capsys: CaptureFixture,
    caplog: LogCaptureFixture,
):
    caplog.set_level(logging.INFO)
    with patch(`"importlib.import_module", function_with_stdout_and_stderr(expected_out, expected_err)):
        ...
    out, err = capsys.readouterr()
    assert not out
    assert not err
    assert expected_out in caplog.text
    assert expected_err in caplog.text
```

Reply:
I still have this on my todo list, but unstacking the latest github mail (last in first out 😄) got me too late in the night yet again :)

Reply:
Thank you for figuring it out!